### PR TITLE
Switch to using midi_interaction in magenta_midi.

### PR DESCRIPTION
**Pull request by adarob**

### CODE REVIEW

**Comment by adarob** [comment]
@hanzorama, agreed that we should add a count-off at some point. Feel free to do it if you have time! I've added clarification about the beats per bar. Thanks.

**Comment by adarob** [comment]
@cghawthorne PTAL

File: magenta/interfaces/midi/README.md [hunk 1]
```diff
@@ -147,61 +151,22 @@ using VPMK and FluidSynth, your command would look like this:
 $ bazel-bin/magenta/interfaces/midi/magenta_midi \
   --input_port="VMPK Output" \
   --output_port="FluidSynth virtual port" \
-  --bundle_file=/tmp/attention_rnn.mag
+  --bundle_file=/tmp/attention_rnn.mag \
+  --phase_bars=4
```
Comment (jesseengel) [inline magenta/interfaces/midi/README.md]: Nit: Do you mean "phrase" bars? Like how long the phrase will be? Phase usually denotes what part of a phrase you would be in, not the length of the phrase.

File: magenta/interfaces/midi/README.md [hunk 2]
```diff
@@ -136,7 +138,9 @@ You can now start the interface with this command, supplying the location of the
 $ bazel-bin/magenta/interfaces/midi/magenta_midi \
   --input_port=<controller port> \
   --output_port=<synthesizer port> \
-  --bundle_file=<bundle_file>
+  --bundle_file=<bundle_file> \
+  --phase_bars=<number of bars per phase or None>
```
Comment (adarob) [inline magenta/interfaces/midi/README.md]: I fixed this by moving things around a bit.

File: magenta/interfaces/midi/README.md [hunk 3]
```diff
@@ -136,7 +138,9 @@ You can now start the interface with this command, supplying the location of the
...
+  --phase_control_number=<control change number of None>
```
Comment (adarob) [inline magenta/interfaces/midi/README.md]: Done.

File: magenta/interfaces/midi/magenta_midi.py [hunk 4]
```diff
@@ -117,476 +70,85 @@
...
+          'specified.')
     return
 
-  if not 0 <= FLAGS.metronome_playback_velocity <= 127:
-    print 'The metronome_playback_velocity must be an integer between 0 and 127'
+  try:
+    bundle = magenta.music.sequence_generator_bundle.read_bundle_file(
+        FLAGS.bundle_file)
+  except magenta.music.sequence_generator_bundle.GeneratorBundleParseException:
+    print 'Failed to prase bundle file: %s' % FLAGS.bundle_file
```
Comment (adarob) [inline magenta/interfaces/midi/magenta_midi.py]: Done

File: magenta/interfaces/midi/README.md (hunk 1 above)
Comment (adarob) [inline magenta/interfaces/midi/README.md]: I meant "phase" as in "call phase" vs "response phase". To remove the confusion, I've changed this to "phrase bars" and used the terms "call stage" and "response stage".

Comment (jesseengel) [inline magenta/interfaces/midi/README.md]: nice

**Comment by hanzorama** [comment]
Will do

**Pull request merged**